### ISSUE TITLE
clock_gettime may require linking librt

### ISSUE THREAD

**Issue by annulen** [annulen/webkit#757]
Handle it like shm_open in WebKit2/CMakeLists.txt

**Comment by annulen** [comment]
```diff
diff --git a/Source/WTF/wtf/PlatformQt.cmake b/Source/WTF/wtf/PlatformQt.cmake
index 684119455b3..0b9de24c895 100644
--- a/Source/WTF/wtf/PlatformQt.cmake
+++ b/Source/WTF/wtf/PlatformQt.cmake
@@ -77,3 +77,14 @@ if (APPLE)
         ${COREFOUNDATION_LIBRARY}
     )
 endif ()
+
+if (UNIX)
+    check_function_exists(clock_gettime CLOCK_GETTIME_EXISTS)
+    if (NOT CLOCK_GETTIME_EXISTS)
+        set(CMAKE_REQUIRED_LIBRARIES rt)
+        check_function_exists(clock_gettime CLOCK_GETTIME_REQUIRES_LIBRT)
+        if (CLOCK_GETTIME_REQUIRES_LIBRT)
+            list(APPEND WTF_LIBRARIES rt)
+        endif ()
+    endif ()
+endif ()

```

**Comment by annulen** [comment]
Fixed in a578d524d55e6b9f58dda4bad293c9f3cc0f78fc
